src/memgraph.cpp

```diff
@@ -26,14 +33,6 @@
 #include <string_view>
 #include <thread>
 
-#include <fmt/core.h>
-#include <fmt/format.h>
-#include <gflags/gflags.h>
-#include <spdlog/common.h>
-#include <spdlog/sinks/daily_file_sink.h>
-#include <spdlog/sinks/dist_sink.h>
-#include <spdlog/sinks/stdout_color_sinks.h>
```

Comment:
The idea/principle here is that 3rd party libs are below std libs (our headers should be in a separated block below the 3rd party libs), that's just the convention which is true for most of the codebase.